defaults/main.yml

```diff
@@ -379,7 +356,7 @@ checkmk_agent__plugin_nginx_servers:
 # .. envvar:: checkmk_agent__git_repo [[[
 #
 # Check_MK agent source repository.
-checkmk_agent__git_repo: 'http://git.mathias-kettner.de/check_mk.git'
+checkmk_agent__git_repo: 'https://git.mathias-kettner.de/check_mk.git'
```

Comment:
Right, I remember, http only. Still their side deployment has some rough edges like that https://git.mathias-kettner.de redirects to http://git.mathias-kettner.de/git/.